src/Telnyx.net/Services/Calls/CallControlApplications/ListCallControlApplicationOptions.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Telnyx.net.Services.Calls.CallControlApplications
+{
+    public class ListCallControlApplicationOptions : ListOptions
+    {
+        public int? PageNumber { get; set; }
```

Comment:
these are now in the base ListOptions Class can be removed (PageNumber and PageSize)

Reply:
Done.